lib/modules/dosomething/dosomething_api/includes/Transformer.php

```diff
@@ -170,11 +166,12 @@ protected function setOffset($page, $count) {
 
   /**
    * @param array $items Collection of item objects retrieved data.
+   * @param string $method Name of method to use on each array item.
    *
    * @return array
    */
-  protected function transformCollection($items) {
-    return array_map(array($this, 'transform'), $items);
+  protected function transformCollection($items, $method = 'transform') {
```

Comment:
Super minor, but calling the variable `method` made me think we're passing an actual function name here. What do you think about renaming it as `$op` ? It's commonly found in the [Drupal universe](https://api.drupal.org/api/drupal/modules%21node%21node.api.php/function/hook_node_access/7).